tests/conftest.py

```diff
@@ -24,6 +24,7 @@
 
 
 def pytest_configure(config):
+    Article.objects.all()
```

Comment:
I also have this in my Mendeley PR (with a comment). Could go out separately. Should probs have a comment though.